### PR TITLE
add keybindings to take snap

### LINKED ISSUE #8: Add keybindings to take snap
I would like to suggest adding keybindings to the capture repository to take a snap. This would make it easier and faster to take snaps, especially for users who frequently take snaps.

Some possible keybindings could be:

- Ctrl+Shift+S
- Ctrl+P
- Cmd+Shift+S (for Mac users)

I believe that adding keybindings to take a snap would be a valuable addition to the capture repository and would make it even more useful for users.

### PR DESCRIPTION
**Pull request by hammadsaedi**
Fixes #8

### CODE REVIEW

**Comment by hammadsaedi** [comment]
@khattakdev check this out, plz

**Review by khattakdev — COMMENTED** [review]
Remove the `.vscode` changes.
We might probably remove the `.vscode` folder from repo or have something as standard in future.

File: package.json [hunk 1]
```diff
@@ -19,6 +19,13 @@
         "command": "capture.snapshot",
         "title": "Capture: Take a snapshot"
       }
+    ],
+    "keybindings": [
+      {
+          "command": "capture.snapshot",
+          "key": "ctrl+alt+p",  
+          "mac": "cmd+alt+p"    
+      }
```
[the remark this answers is not in the document]
Comment (hammadsaedi) [reply package.json:28]: what should I do now? 

Means what about this PR? How can I edit this? [noob question]

[the remark this answers is not in the document]
Comment (khattakdev) [reply package.json:28]: Simply do the changes in the branch locally, commit them and pushed them like you did before.
It will update this PR.

**Comment by hammadsaedi** [comment]
@khattakdev
- added `editorHasSelection`
- removed the `.vscode` changes.

**Review by khattakdev — APPROVED** [review]

**Pull request merged**